src/NuGet.Services.AzureSearch/Models/BaseMetadataDocument.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Azure.Search;
+
+namespace NuGet.Services.AzureSearch
+{
+    public abstract class BaseMetadataDocument : KeyedDocument
```

Comment:
Some documentation for the new classes?..

Reply:
Yeah, good idea.